client/src/index.jsx

```diff
@@ -1,12 +1,163 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import './styles.css';
+
+class App extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {number: 1};
```

Comment:
"number" is extremely vague. What is it, why does it start at 1?

Reply:
I'd actually changed number to 0 since. It's the number of test sightings that are displayed. So every time you click "add sighting, number increases by one, and one more sighting is displayed. Can rename variable to sightingCount or something like that.

Reply:
But why is this necessary?
You have the number of sightings with `[].length`

Reply:
In production yes, you would have x sightings in your list and could use [].length At this point I was trying to make it so that clicking the button autocreated a new test listing, so [].length should be 0 at the start each time. I think.